src/platform/functions/index.js

```diff
@@ -12,11 +12,20 @@ exports.scrapeEventbrite = functions
     timeoutSeconds: 60,
     memory: '1GB',
   })
-  .https.onRequest(async (_, res) => {
+  .https.onRequest(async (req, res) => {
     functions.logger.info('Starting to scrape...');
     let eventsArray = [];
     let collectiveEventsArray = [];
 
+    let city = '';
+    if (req.method === 'GET') {
+      if (req.query.city.length !== 0 && typeof req.query.city !== 'undefined') {
+        city = req.query.city;
+        functions.logger.info('City: ' + city);
+      } else {
+        functions.logger.error('No city name provided');
```

Comment:
```suggestion
        return functions.logger.error('No city name provided');
```
In the case where no city is provided, we will just stop the function

Reply:
done!